tasks/winbuildscripts/unittests.ps1

```diff
@@ -54,7 +54,7 @@ if($err -ne 0){
     Write-Host -ForegroundColor Red "Agent build failed $err"
     [Environment]::Exit($err)
 }
-& inv -e test --junit-tar="$Env:JUNIT_TAR" --race --profile --rerun-fails=2 --coverage --cpus 8 --python-runtimes="$Env:PY_RUNTIMES" --python-home-2=$Env:Python2_ROOT_DIR --python-home-3=$Env:Python3_ROOT_DIR --save-result-json C:\mnt\$test_output_file $Env:EXTRA_OPTS --build-stdlib $TEST_WASHER_FLAG
+& inv -e test --junit-tar="$Env:JUNIT_TAR" --race --profile --rerun-fails=2 --coverage --cpus 8 --python-home-2=$Env:Python2_ROOT_DIR --python-home-3=$Env:Python3_ROOT_DIR --save-result-json C:\mnt\$test_output_file $Env:EXTRA_OPTS --build-stdlib $TEST_WASHER_FLAG
```

Comment:
Seems to be referencing a PY2 related dir ` --python-home-2=$Env:Python2_ROOT_DIR`

Reply:
Yep, those are planned for a later iterations, `--python-runtimes` was enough changes :sweat_smile: 

That being said I suspect `PythonX_ROOT_DIR` serves no purpose. My understanding is that it's meant to force a version being used by RtLoader, but it is ignored by cmake.